cmd/ai-proxy/bootstrap.yml

```diff
@@ -10,6 +10,7 @@ erda.app.ai-proxy:
   routesRef: conf/routes.yml
   providersRef: conf/providers.yml
   logLevel: ${ env.LOG_LEVEL:debug }
+  erdaOpenapi: ${ env.ERDA_OPENAPI:https://openapi.erda.cloud }
```

Comment:
why is the format not like this: `erdaOpenapi: ${ERDA_OPENAPI:https://openapi.erda.cloud}`